http/swagger.yml

```diff
@@ -4564,6 +4566,28 @@ components:
           type: string
         dialect:
           $ref: "#/components/schemas/Dialect"
+    FileASTNode:
+      description: flux File AST node
+      type: object
+      properties:
+        type:
+          description: type of AST node
+          type: string
+          enum:
+            - File
```

Comment:
Is this enumeration validated on query (that it can only be `File`)?  I didn't see it... perhaps it is done in flux unmarshaling code?

Reply:
Yes this is handled in the unmarshaling code.